Jenkinsfile

```diff
@@ -215,6 +215,116 @@ pipeline {
+                    withCredentials([usernamePassword(credentialsId: ARTIFACTORY_CREDENTIALS_ID, usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
+                        // TODO: Consider using tooling like artifactory-download-spec to get license.zip. Post-Infrastructure migration answer.
+                        sh "mkdir -p licenses && cd licenses && curl -s -o zowe_licenses_full.zip $ZOWE_LICENSE_ZIP_URL"
+                        sh "./scripts/npm_login.sh $USERNAME $PASSWORD \"$ARTIFACTORY_EMAIL\" '--registry=${ZOWE_ARTIFACTORY_URL} --scope=@zowe'"
+                    }
+                    sh "npm install jsonfile"
+
+                    sh "npm pack @zowe/core-for-zowe-sdk@zowe-v1-lts"
+                    sh "npm pack @zowe/provisioning-for-zowe-sdk@zowe-v1-lts"
+                    sh "npm pack @zowe/zos-console-for-zowe-sdk@zowe-v1-lts"
+                    sh "npm pack @zowe/zos-files-for-zowe-sdk@zowe-v1-lts"
+                    sh "npm pack @zowe/zos-jobs-for-zowe-sdk@zowe-v1-lts"
+                    sh "npm pack @zowe/zos-tso-for-zowe-sdk@zowe-v1-lts"
+                    sh "npm pack @zowe/zos-uss-for-zowe-sdk@zowe-v1-lts"
+                    sh "npm pack @zowe/zos-workflows-for-zowe-sdk@zowe-v1-lts"
+                    sh "npm pack @zowe/zosmf-for-zowe-sdk@zowe-v1-lts"
+
+                    sh "./scripts/repackage_bundle.sh *.tgz" // Outputs a zowe-cli-package.zip
+                    sh "mv zowe-cli-package.zip zowe-nodejs-sdk-${ZOWE_CLI_BUNDLE_VERSION}.zip"
+
+                    archiveArtifacts artifacts: "zowe-nodejs-sdk-${ZOWE_CLI_BUNDLE_VERSION}.zip"
+
+                    // Remove all tgzs after bundle is archived
+                    sh "rm -f *.tgz"
+                }
+            }
+        }
+        /************************************************************************
+         * STAGE
+         * -----
+         * Build Zowe Python SDK Bundle
+         *
+         * TIMEOUT
+         * -------
+         * 10 Minutes
+         *
+         * EXECUTION CONDITIONS
+         * --------------------
+         * - Always
+         *
+         * DECRIPTION
+         * ----------
+         * Gets the latest version of the Zowe Python SDK package from pypi.org
+         *
+         * OUTPUTS
+         * -------
+         * A Zowe Python SDK Archive.
+         ************************************************************************/
+        stage('Create Zowe Python SDK Bundle') {
+            when {
+                allOf {
+                    expression {
+                        return BRANCH_NAME.equals(MASTER_BRANCH)
+                    }
+                }
+            }
+            steps {
+                timeout(time: 10, unit: 'MINUTES') {
+                    // Install python if needed
+                    // sh "apt update && apt install python3-pip || exit 0"
+                    // sh "yum -y update && yum -y install python3-pip || exit 0"

```

Comment:
Waiting for confirmation.

Reply:
Hey @MarkAckert,
Any advice here?

Reply:
Image is updated with python3-pip - give it a run and let me know if you need any other packages installed